src/routes/transactions/mappers/common/transactions.mapper.ts

```diff
@@ -0,0 +1,219 @@
+import { Injectable } from '@nestjs/common';
+import {
+  isCreationTransaction,
+  isEthereumTransaction,
+  isModuleTransaction,
+  isMultisigTransaction,
+  Transaction as TransactionDomain,
+} from '../../../../domain/safe/entities/transaction.entity';
+import { Safe } from '../../../../domain/safe/entities/safe.entity';
+import { ModuleTransactionMapper } from '../module-transactions/module-transaction.mapper';
+import { MultisigTransactionMapper } from '../multisig-transactions/multisig-transaction.mapper';
+import { MultisigTransaction } from '../../../../domain/safe/entities/multisig-transaction.entity';
+import { ModuleTransaction } from '../../../../domain/safe/entities/module-transaction.entity';
+import { EthereumTransaction } from '../../../../domain/safe/entities/ethereum-transaction.entity';
+import { groupBy } from 'lodash';
+import { IncomingTransferMapper } from '../transfers/transfer.mapper';
+import { TransactionItem } from '../../entities/transaction-item.entity';
+import { DateLabel } from '../../entities/date-label.entity';
+import { Transfer } from '../../../../domain/safe/entities/transfer.entity';
+import { CreationTransaction } from '../../../../domain/safe/entities/creation-transaction.entity';
+import { CreationTransactionMapper } from '../creation-transaction/creation-transaction.mapper';
+
+class TransactionDomainGroup {
+  timestamp: number;
+  transactions: (
+    | MultisigTransaction
+    | ModuleTransaction
+    | EthereumTransaction
+    | CreationTransaction
+  )[];
+}
+
+@Injectable()
+export class TransactionsMapper {
```

Comment:
I'd say this mapper is not so generic, it contains specific logic coupled to the transactions history endpoint (`DateLabels`), so I'm thinking maybe this should be named something like `HistoryItemMapper`, `TransactionsHistoryItemMapper`, `TransactionsHistoryMapper`... and maybe it doesn't make a lot of sense to have it in the `common` folder. What do you think @fmrsabino @moisses89?

Reply:
Agree with the third one :) and move to mappers root.

Reply:
@hectorgomezv 6251e27dc680bce987ff0e6688054b33644c5805 what do you think?

Reply:
Awesome, thanks 🚀